src/util/storagePool.tsx

```diff
@@ -25,6 +25,10 @@ export const storagePoolFormFieldToPayloadName: Record<string, string> = {
   cephfs_osd_pg_num: "cephfs.osd_pg_num",
   cephfs_path: "cephfs.path",
   cephfs_user_name: "cephfs.user.name",
+  cephobject_radosgw_endpoint: "cephobject.radosgw.endpoint",
+  cephobject_cluster_name: "cephobject.cluster_name",
+  cephobject_user_name: "cephobject.user.name",
+  cephobject_bucket_name_prefix: "cephObject.bucket.name_prefix",
```

Comment:
```suggestion
  cephobject_bucket_name_prefix: "cephobject.bucket.name_prefix",
```
This is case-sensitive and prevents the help text for the field to show up. I think this also prevents the field from being saved or preseeded on edit.